internal/audit/cluster_checks.go

```diff
@@ -214,3 +218,58 @@ func makeCheckClusterHighHAAssets(haAssetsThreshold int) checkFunc {
 		return Pass, nil
 	}
 }
+
+func checkClusterNamesForWhitespace(reader *archive.Reader, examples *ExamplesCollection) (Outcome, error) {
+
+	for _, clusterName := range reader.GetClusterNames() {
+		if strings.ContainsFunc(clusterName, unicode.IsSpace) {
+			examples.add("Cluster name '%s' contains whitespace", clusterName)
```

Comment:
This is an example printed after the check name, so 'contains whitespace is redundant' it should just be:

`examples.add("Cluster: '%s', clusterName)`